<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>noteWindow</class>
 <widget class="QMainWindow" name="noteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Planner</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resources/apple.PNG</normaloff>../resources/apple.PNG</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="1">
         <widget class="QPushButton" name="tasksButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255,255,255,70);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../resources/BBFB64C24270F0A6CC0C615A226C7EA1.png</normaloff>../resources/BBFB64C24270F0A6CC0C615A226C7EA1.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" rowspan="7">
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255,255,255,70);</string>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="community">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="list" name="listView_2">
              <property name="styleSheet">
               <string notr="true"> QPalette pa(this-&gt;palette());

    QImage img = QImage(&quot;../resources/nbg.PNG&quot;);
    img = img.scaled(this-&gt;size());


    QBrush *pic = new QBrush(img);

    pa.setBrush(QPalette::Window,*pic);

    //this-&gt;setAutoFillBackground(true);
    this-&gt;setPalette(pa);</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="myNote">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="0">
             <widget class="QPushButton" name="addnote">
              <property name="styleSheet">
               <string notr="true">background-color: rgba(255,255,255,70);</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../resources/50AC34918CBBAA2FBC29802F50C99A66.png</normaloff>../resources/50AC34918CBBAA2FBC29802F50C99A66.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>338</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="list" name="listView">
              <property name="styleSheet">
               <string notr="true">background-color: rgba(255,255,255,70);</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="6" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>358</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="noteButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255,255,255,70);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../resources/6D14136C9A107D92EFE77D75B7035FB5.png</normaloff>../resources/6D14136C9A107D92EFE77D75B7035FB5.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="more">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255,255,255,70);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../resources/73AD02A873A289F08630F02E631A49BC.png</normaloff>../resources/73AD02A873A289F08630F02E631A49BC.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="tomatoButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255,255,255,70);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../resources/7254AC58E85FA5C1CE8CA49EB6064CF4.png</normaloff>../resources/7254AC58E85FA5C1CE8CA49EB6064CF4.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="comButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255,255,255,70);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../resources/5F770C89EAA1E50FF18563ED558A3484.png</normaloff>../resources/5F770C89EAA1E50FF18563ED558A3484.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>标题：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255,255,255,70);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="5">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>内容：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" rowspan="2">
           <widget class="QTextEdit" name="textEdit">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255,255,255,70);</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>是否公开：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton">
            <property name="text">
             <string>私有</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_2">
            <property name="text">
             <string>公开</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1" colspan="4">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>350</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>169</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="save">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255,255,255,70);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../resources/37CF57182DDEB6469851836D20F00638.png</normaloff>../resources/37CF57182DDEB6469851836D20F00638.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>70</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>21</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="cancel">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255,255,255,70);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../resources/EC5CB6F3435B6BC9B80A3D2CBC3993B4.png</normaloff>../resources/EC5CB6F3435B6BC9B80A3D2CBC3993B4.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>70</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>148</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="table" name="tableView"/>
        </item>
        <item row="1" column="0">
         <widget class="table" name="tableView_2"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>table</class>
   <extends>QTableView</extends>
   <header>table.h</header>
  </customwidget>
  <customwidget>
   <class>list</class>
   <extends>QListView</extends>
   <header>list.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
